cli/cmd/sudo/superuser/remove.go

```diff
@@ -0,0 +1,54 @@
+package superuser
+
+import (
+	"fmt"
+
+	"github.com/rilldata/rill/cli/pkg/cmdutil"
+	"github.com/rilldata/rill/cli/pkg/config"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"github.com/spf13/cobra"
+)
+
+func RemoveCmd(cfg *config.Config) *cobra.Command {
+	var email string
+	removeCmd := &cobra.Command{
+		Use:   "remove <email>",
+		Args:  cobra.MaximumNArgs(1),
```

Comment:
Must be exact

Reply:
Do we need this with flags? i mean `--email` ? though of adding that. If not we can have it as exact.